pkg/backup/item_collector.go

```diff
@@ -289,7 +289,7 @@ func (r *itemCollector) getResourceItems(log logrus.FieldLogger, gv schema.Group
 			return nil, errors.WithStack(err)
 		}
 
-		items := r.backupNamespaces(unstructuredList, namespacesToList, gr, preferredGVR, log)
+		items := r.backupNamespaces(unstructuredList, r.backupRequest.NamespaceIncludesExcludes, gr, preferredGVR, log)
```

Comment:
if `namespacesToList` is not used in this chunk we should move it closer to the chunk it's used.
Same applies to `clusterScoped`

Reply:
Modified.